src/main/java/io/confluent/idesidecar/restapi/proxy/ProxyContext.java

```diff
@@ -25,7 +25,7 @@ public class ProxyContext {
   final Buffer requestBody;
   final Map<String, String> requestPathParams;
 
-  final @Nullable String connectionId;
+  final String connectionId;
```

Comment:
Why this change? Are we changing the contract, or is this a correction (e.g., the field is not really allowed to be null)?

Reply:
This was something I'd added in, probably to make a test pass, but no longer needed, so I reverted it to the prior state